packages/packages.json

```diff
@@ -42,6 +42,7 @@
         "skill/valory/transaction_settlement_abci/0.1.0": "bafybeia7rzsbea3ch4gcafyp3z6uvqh4npws2xpdwbkkdbrqqpjops7nui",
         "skill/valory/termination_abci/0.1.0": "bafybeigqpij2sgrpnilqjljfciixop4fldq5qceixc7534q6af4potdmdm",
         "contract/valory/gnosis_safe_proxy_factory/0.1.0": "bafybeid6glyjikjxmefwmhn62cxiofophegjmg2z5vqqsvk6tmyunwc274",
-        "contract/valory/gnosis_safe/0.1.0": "bafybeih6d3vxz3jlgodxm5b2qcwsmansqj4xobuyd6hjnhzremuvd65yrm"
+        "contract/valory/gnosis_safe/0.1.0": "bafybeih6d3vxz3jlgodxm5b2qcwsmansqj4xobuyd6hjnhzremuvd65yrm",
+        "contract/valory/mech/0.1.0": "bafybeiddc6pgurpyja2k64wzsb3jgcvl254s7rplgt5iooftsyfalqlbfq"
```

Comment:
As mentioned above, this will need to be a dev package.

That is because you are using multisend to send the requests and you need to parse the logs differently than the third-party contract you utilized from the trader service. More specifically, you will need to expect as many logs as the number of requests you send [here](https://github.com/valory-xyz/trader/blob/86969cf3d203a2a68acee4912f22969db5542286/packages/valory/contracts/mech/contract.py#L68-L97). The method currently expects a single log per tx hash.

Moreover, I would suggest creating a new method for this instead of editing the existing method, so that the contract package can become a third-party dependency in the trader service.